src/lib/components/chat/ChatIntroduction.svelte

```diff
@@ -10,6 +10,7 @@
 	import ModelCardMetadata from "../ModelCardMetadata.svelte";
 	import type { LayoutData } from "../../../routes/$types";
 	import { findCurrentModel } from "$lib/utils/models";
+	import { env } from "$env/dynamic/public";
```

Comment:
Can you put the default announcement in the `.env` file? Then you should be able to import `PUBLIC_ANNOUNCEMENT_BANNERS` form `$env/static/public`

Reply:
Sure thing. Thanks.